src/components/Layout/HomeContent.js

```diff
@@ -415,7 +435,11 @@ export function HomeContent() {
           <div className="max-w-7xl mx-auto flex flex-col lg:flex-row px-5">
             <div className="max-w-3xl lg:max-w-7xl gap-5 flex flex-col lg:flex-row lg:px-5">
               <div className="w-full lg:w-6/12 max-w-3xl flex flex-col items-start justify-start lg:ps-5 lg:pe-10">
-                <Header>새로운 기능이 준비되면 업그레이드 하기</Header>
+                <Header>
+                  새로운 기능에 맞춰
+                  <Br />
+                  업그레이드 하기
```

Comment:
```suggestion
새로운 기능에 맞춰 <Br breakPointPrefix='xl' /> 업그레이드 하기
```

여기에 props로 `'xl'`을 전달합니다.